torch/lib/THCUNN/generic/SpatialClassNLLCriterion.cu

```diff
@@ -148,4 +231,6 @@ void THNN_(SpatialClassNLLCriterion_updateGradInput)(
   THCTensor_(free)(state, input);
 }
 
+#undef TO_DEVICE
```

Comment:
What's this?

Reply:
Forgot to remove that, thanks for pointing it out!